### DISCUSSION TITLE
希望能增加手机localsend后台自动弹出/后台自动接收文件的功能，这样每次传输文件的时候手机就不需要打开app界面了

### DISCUSSION THREAD

**Discussion by BluedL** [localsend/localsend#989] [L-简体中文]
每次传输文件都要打开手机的界面，真的很不方便

**Comment by beker888** [comment]
强烈支持楼主的建议！不知道这是不是要增加后台刷新功能实现。

**Comment by bearcloney** [comment]
这牵扯到各大手机厂商的app后台保活，感觉不如期待酷安老哥做magisk插件的可能性大

**Comment by zazitufu** [comment]
万一有人同局域网send一些莫名其妙的东西呢？

**Reply by bearcloney** [reply]
app不是本身就有快速保存开关吗，你要局域网那么多人，你担心可以不开。
我一般局域网最多家人朋友，根本不担心，更何况他们也不用这软件

**Reply by zazitufu** [reply]
我在家里和在公司都有用。在家里基本打开就好了。

但是在公司，希望还是有一个类似验证码的东西，这样接收端可以设定，知道验证码的可以直接接收，如果不知道验证码的设备，发送过来就需要手动确认。

**Reply by bearcloney** [reply]
确实是个问题，不过我认为有这个需求的人可能意外的没那么多，毕竟本质上只需要在首页上切换以下快速保存开关
如果要实现，可能要通过mac地址判断设备？但是mac有些默认会伪装而且也不知道普通设备有没有权利查看局域网内的设备mac，
靠ip地址判断，考虑到DHCP租期一个星期可能又得重设。用静态ip公司人多，或许一不小心就复用了别人的ip

实在不行工位下面自己装个中继路由开dhcp也不是不行，，吧

**Comment by qiukun39** [comment]
支持这个功能，我想让这个软件代替微信实现手机 电脑传图传文件交互，但是每次电脑传送到手机，先要把手机APP打开，，电脑上附近的设备也不是立刻显示，还要刷新扫描，才显示设备，有些拖沓
，希望增加让附近设备一致在线的功能

**Comment by CSZCTZY** [comment]
做个快捷开关丢下拉栏里就行了，留存后台肯定耗电

**Comment by bearcloney** [comment]
其实在原生安卓LineageOS，即使软件运行在后台也可以直接传文件，我怀疑可能是各大厂商的rom优化问题

**Comment by zwc456baby** [comment]
希望能增加一个选项，开启一个 通知栏的 前台服务。总是要打开app才能传输。某些场景不够快速，电脑有后台服务的选项，手机传电脑很方便。

**Comment by miaozilong** [comment]
2025年了，还是没有这个功能呢。

**Comment by windyy0** [comment]
感觉至少能实现传文件时候可以后台传吧，每次传大文件，手机就必须放在前台，不能干其他事了

**Comment by xiu58585** [comment]
是啊 强烈建议后台保活，每次都要退出app， 重新进去才能刷新到其他设备或者被搜索到，有点太麻烦了...

**Comment by I21b** [comment]
与 #141 相同
